spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+RSpec.describe "ingredients index" do 
+  before :each do 
+    @chicken = Ingredient.create!(name: "Chicken", cost: 5)
+    @tortillas = Ingredient.create!(name: "Tortillas", cost: 2)
+    @beef = Ingredient.create!(name: "Beef", cost: 4)
+    @pasta = Ingredient.create!(name: "Pasta", cost: 1)
+  end
+  describe "#index" do 
+    it "display name and cost of all ingredients" do 
+      visit "/ingredients"
+
+      expect(page).to have_content("All Ingredients Available")
+
+      expect(page).to have_content(@chicken.name)
```

Comment:
This test is good, but it could be more robust with the use of within blocks to verify that, for example, the chicken price is within the section for chicken, and is not just a floating `5` integer somewhere else on the page.